Fix FTBFS

../../components/viz/host/gpu_host_impl.cc -o obj/components/viz/host/host/gpu_host_impl.o
../../components/viz/host/gpu_host_impl.cc:30:10: fatal error: 'gpu/webgpu/dawn_commit_hash.h' file not found
   30 | #include "gpu/webgpu/dawn_commit_hash.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- /dev/null
+++ b/gpu/webgpu/dawn_commit_hash.h
@@ -0,0 +1,8 @@
+/* Generated by lastchange.py, do not edit.*/
+
+#ifndef GPU_WEBGPU_DAWN_COMMIT_HASH_H_
+#define GPU_WEBGPU_DAWN_COMMIT_HASH_H_
+
+#define DAWN_COMMIT_HASH "cee9cb0d67e749bf42f5e90cb3b8a6f525dbb920"
+
+#endif  // GPU_WEBGPU_DAWN_COMMIT_HASH_H_
